tree/30_generic_methods/file_ensure_key_value.cf

```diff
@@ -0,0 +1,47 @@
+#####################################################################################
+# Copyright 2016 Normation SAS
+#####################################################################################
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, Version 3.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+#####################################################################################
+
+# @name File ensure key -> value present
+# @description Ensure that the file contains a pair of "key separator value", with arbitrary separator between each key and its value. It the key is already present, it will change the value associated to this key.
```

Comment:
This is rather a long description. Could you split it out into a single line summary in @description and a more detailed explanation in @documentation? See https://github.com/Normation/ncf/blob/master/tree/30_generic_methods/variable_dict.cf#L20 for an example.